packages/context/README.md

```diff
@@ -1 +1,59 @@
 # @loopback/context
+
+Loopback uses Context to manage state and dependencies in your application.
+
+## Overview
+
+- An abstraction of all state and dependencies in your application.
+- Context is what LB uses to "manage" everything.
+- A global registry for anything/everything in your app (all configs, state, dependencies, classes, etc).
+- An IoC container used to inject dependencies into your code.
```

Comment:
This section should be a simple paragraph (not bullet list) describing everything above.

LoopBack implements the concept of a Context to represent a global registry in your application to manage an application's state and dependencies...etc